test/ssc_test/cmod_pvwattsv5_test.cpp

```diff
@@ -180,4 +179,17 @@ TEST_F(CMPvwattsV5Integration_cmod_pvwattsv5, singleTS) {
     EXPECT_NEAR(val, 106459, 1);
     ssc_data_get_number(data, "ac", &val);
     EXPECT_NEAR(val, 100579, 1);
+
+    // add some shading
+    ssc_data_set_number(data, "shaded_percent", 50);
+    EXPECT_TRUE(ssc_module_exec(mod, data));
+    ssc_data_get_number(data, "poa", &val);
+    EXPECT_NEAR(val, 140.05, .1);
+    ssc_data_get_number(data, "tcell", &val);
+    EXPECT_NEAR(val, 13.36, .1);
+    ssc_data_get_number(data, "dc", &val);
+    EXPECT_NEAR(val, 106341, 1);
+    ssc_data_get_number(data, "ac", &val);
+    EXPECT_NEAR(val, 100463, 1);
```

Comment:
Added some testing with shading. It didn't decrease the power output as much as I thought it would, but then again I'm not a solar guy so maybe I don't have as good a sense of what the results _should_ be.

Reply:
Thanks! I think this makes sense because the solar data here has very little beam irradiance compared to diffuse. The shaded_percent only gets applied to the beam so overall there's little effect. This and the (1-shaded_percent) is consistent with `shading_factor_calculator`'s `shading:timestep` with default `shading:string_option == -1`.